src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/HdcedCalculator.kt

```diff
@@ -105,38 +168,59 @@ class HdcedCalculator(val hdcedConfiguration: HdcedConfiguration) {
     }
   }
 
+  private fun getAddedDays(sentenceCalculation: SentenceCalculation) = sentenceCalculation.calculatedTotalAddedDays
+    .plus(sentenceCalculation.calculatedTotalAwardedDays)
+    .minus(sentenceCalculation.unusedAdaDays)
+
+  private fun getDeductedDays(sentenceCalculation: SentenceCalculation) =
+    sentenceCalculation.calculatedTotalDeductedDays.toLong()
+
   private fun calculateHdcedMinimumCustodialPeriod(
+    sentence: CalculableSentence,
     sentenceCalculation: SentenceCalculation,
-    sentenceDate: LocalDate,
     parentRule: CalculationRule,
-    addedDays: Long,
+    params: Hdced4Params,
```

Comment:
Perhaps next iteration is to rename this?

Reply:
No - I should fix that too!

Reply:
Done - I think